exercises/gigasecond/src/example/java/Gigasecond.java

```diff
@@ -1,19 +1,20 @@
 import java.time.LocalDateTime;
 import java.time.LocalDate;
 
-public class Gigasecond {
+class Gigasecond {
 
-    private LocalDateTime birthDateTime;
+    private LocalDateTime initialDateTime;
```

Comment:
I agree that calling it `initialDateTime` makes the class more general and might be nicer. But calling it `birthDateTime` lines up more with the description of the exercise and might make it easier for people solving it to understand what they need to do? I don't particularly mind though :)